api/src/main.rs

```diff
@@ -17,25 +16,11 @@ async fn main() {
     env_logger::init();
     log::info!("Starting service");
 
-    let configuration = {
-        dotenv().ok();
-        match Configuration::init_from_env() {
-            Ok(val) => val,
-            Err(e) => {
-                log::error!("{}", e);
-                log::error!("Terminating because of previous error.");
-                process::exit(1);
-            }
-        }
-    };
-    match CONFIGURATION.set(configuration) {
-        Ok(()) => log::info!("Successfully provided global service configuration"),
-        Err(_) => {
-            log::error!("Failed to provide global service configuration");
-            log::error!("Terminating because of previous error.");
-            process::exit(1);
-        }
-    }
+    dotenv().ok();
+    let conf = Configuration::init_from_env().expect("Error getting configuration from env");
```

Comment:
Here we are losing the error output of the `envconfig` library about why the initialization failed.

Reply:
Actually not, since `.expect(...)` appends the error message (see [docs](https://doc.rust-lang.org/std/result/enum.Result.html#method.expect)).